crates/router/src/core/payouts/helpers.rs

```diff
@@ -676,14 +671,9 @@ pub async fn save_payout_data_to_locker(
         locker_req.update_requestor_card_reference(Some(card_reference.to_string()));
 
         // Store in locker
-        let stored_resp = cards::add_card_to_hs_locker(
-            state,
-            &locker_req,
-            customer_id,
-            api_enums::LockerChoice::HyperswitchCardVault,
-        )
-        .await
-        .change_context(errors::ApiErrorResponse::InternalServerError);
+        let stored_resp = cards::add_card_to_hs_locker(state, &locker_req, customer_id)
```

Comment:
```suggestion
        let stored_resp = cards::add_card_to_vault(state, &locker_req, customer_id)
```
Can we change the function naming across all the files and make them meaningful as right now I can see somewhere hs being used and others using basilisk.